morse_talk/gui_func.py

```diff
@@ -0,0 +1,73 @@
+from tkinter import *
+import time
+import morse_talk as mtalk
+import sys
+
+dc = ''
+cd = ''
+
+
+def gui():
+    def encode():
+        global dc
+        x = e1.get()
+        q = mtalk.encode(x)
+        if q != dc:
+            dc = q
+            e3.delete(0, END)
+            e3.insert(0, q)
+        e3.after(500, encode)
+
+    def decode():
+        global cd
+        global y
+        if e2.get() == '':
+            y = "a"
+        else:
+            y = e2.get()
+        z = ''
+        r = ""
+        if y[-1] == ' ':
+            r = mtalk.decode(y)
+            if r != cd:
+                cd = r
+                e4.delete(0, END)
+                e4.insert(0, r)
+        e4.after(500, decode)
+
+    dc = ''
+    cd = ''
+    y = "a"
+    v='Courier', 20, 'bold'
```

Comment:
PEP8 (space before and after `=`)

Why such names for variables? Don't you think that `font1` and `font2` won't be more expressive?
(or `font_label` and `font_entry`)